app/views/layouts/application.html.slim

```diff
@@ -68,12 +74,13 @@ html
             .alert.alert-danger= alert
       .row.py-3.main-content
         = yield
-    .footer.dontprint
+    .footer.dontprint.bg-dark.text-light
       .container
         .row
-          .col-4.nav-item
-            => link_to 'Support NISEI on Patreon', 'https://www.patreon.com/join/NISEI'
-          .col-4.nav-item.text-center
-           | 🐍 Maintained by NISEI. Created by Johno. 🐍 
-          .col-4.nav-item.text-right
+          .col-3
```

Comment:
why these changes?  Typically, things have been in a bootstrap 12-column width, but this shifts this up.

The layout shifted on the bottom navbar:
old:
![Screenshot 2022-02-05 12 48 09 PM](https://user-images.githubusercontent.com/396562/152654849-d4fedade-4899-4a78-972c-ea3e6dd62b44.png)

new:
![Screenshot 2022-02-05 12 44 38 PM](https://user-images.githubusercontent.com/396562/152654822-e4ec1e22-4eea-43f7-a316-99c92a515026.png)

Reply:
I was trying to clean up the navbar and footer as I was poking around to use minimal custom classes. Then I got carried away... 😅 

I'll pull these changes out and save them for a bigger UI cobra improvement project!